docs/contributing.rst

```diff
@@ -2,22 +2,25 @@
 Contributing
 ############
 
-Help is always appreciated!
+Help is always appreciated and there are plenty of options how you can contribute to Solidity.
 
-To get started, you can try :ref:`building-from-source` in order to familiarize
-yourself with the components of Solidity and the build process. Also, it may be
-useful to become well-versed at writing smart-contracts in Solidity.
-
-In particular, we need help in the following areas:
+In particular, we appreciate support in the following areas:
 
-* Improving the documentation
-* Responding to questions from other users on `StackExchange
-  <https://ethereum.stackexchange.com>`_ and the `Solidity Gitter
-  <https://gitter.im/ethereum/solidity>`_
+* Reporting issues.
 * Fixing and responding to `Solidity's GitHub issues
   <https://github.com/ethereum/solidity/issues>`_, especially those tagged as
   `good first issue <https://github.com/ethereum/solidity/labels/good%20first%20issue>`_ which are
   meant as introductory issues for external contributors.
+* Improving the documentation.
+* Translating the documentation into more languages.
+* Responding to questions from other users on `StackExchange
+  <https://ethereum.stackexchange.com>`_ and the `Solidity Gitter Chat
+  <https://gitter.im/ethereum/solidity>`_.
+* Getting involved in the language design process by joining language design calls, proposing language changes or new features and providing feedback.
+
+To get started, you can try :ref:`building-from-source` in order to familiarize
```

Comment:
```suggestion
If you wish to contribute code, you can start with :ref:`building-from-source` in order to familiarize
```
Something like that? Since the bullets above wouldn't necessarily require this step